patchwork-events-world/src/main/resources/patchwork-events-world.mixins.json

```diff
@@ -4,7 +4,10 @@
   "compatibilityLevel": "JAVA_8",
   "mixins": [
     "MixinServerPlayerInteractionManager",
-    "MixinThreadedAnvilChunkStorage"
+    "MixinThreadedAnvilChunkStorage",
+    "MixinClientWorld",
+    "MixinServerWorld",
+    "MixinMinecraftServer"
```

Comment:
... oops, i forgot to add all of the new mixins here, which means that it's even less tested than I thought. *sigh*

I'll come back to this later. I've already been working on this for hours in a row today.